xds/src/test/java/io/grpc/xds/internal/sds/SecretVolumeSslContextProviderTest.java

```diff
@@ -419,8 +404,22 @@ static void doChecksOnSslContext(boolean server, SslContext sslContext,
   /**
    * Helper method to build DownstreamTlsContext for above tests. Called from other classes as well.
    */
-  static DownstreamTlsContext buildDownstreamTlsContextFromFilenames(
-      String privateKey, String certChain, String trustCa) {
+  public static DownstreamTlsContext buildDownstreamTlsContextFromFilenames(
```

Comment:
should be moved to utility class if shared in many tests.

Reply:
For file-based credentials (as the name suggests), this can be considered as the utility class. Are you thinking of one utility class to have all these methods?